src/manager.cpp

```diff
@@ -202,10 +203,21 @@ void Manager::getRsyncCmd(RsyncMode mode,
     else if ((dataSyncCfg._includeList.has_value()) && (srcPath.empty()))
     {
         // Configure the paths in include List as SRC paths
-        auto appendToCmd = [&cmd](const auto& path) {
-            cmd.append(" "s + path.string());
+        auto exists_now = [](const fs::path& p) {
+            std::error_code ec;
+            return fs::exists(p, ec);
         };
-        std::ranges::for_each(dataSyncCfg._includeList.value(), appendToCmd);
+
+        if (std::ranges::none_of(dataSyncCfg._includeList.value(), exists_now))
+        {
+            lg2::info(
+                "IncludeList: none of the configured source paths exist, skipping rsync");
+            haveIncList = true;
+        }
+
+        std::ranges::for_each(
```

Comment:
I think you are checking existence twice so it could tweaked as like below
```
// Paths that do not yet exist will be synced once they are created; otherwise, the sync will fail.

auto existPath = [](const fs::path& p) {
    std::error_code ec;
    return fs::exists(p, ec);
};

bool anyExist = false;
std::ranges::for_each(dataSyncCfg._includeList.value() | std::views::filter(existPath),
    [&](const fs::path& p) {
        cmd.append(" "s + p.string());
        anyExist = true;
    });

if (!anyExist)
{
    lg2::debug("None of the configured include paths are exist, skipping rsync");
    cmd.clear();
    return
}
```